server/base/src/main/java/org/apache/accumulo/server/ServerConstants.java

```diff
@@ -44,11 +44,10 @@
   public static final String INSTANCE_ID_DIR = "instance_id";
 
   /**
-   * current version (3) reflects additional namespace operations (ACCUMULO-802) in version
-   * 1.6.0<br>
-   * (versions should never be negative)
+   * Accumulo servers will only communicate with each other when this is the same. Bumped for 2.0 to
+   * prevent 1.9 and 2.0 servers from communicating.
```

Comment:
Could mention `This is not a client-to-server wire version`

Reply:
Instead of a comment I decided to move it to the only place it used, SystemToken.  I think this will make things much clearer.